.env.example

```diff
@@ -0,0 +1,2 @@
+REACT_APP_YOUTUBE_BASE_URL=https://youtube.googleapis.com/youtube/v3
```

Comment:
Nice!
Thank you for using best practices and taking the time to use env variables